docs/source/cloud-setup/policy.rst

```diff
@@ -62,34 +83,107 @@ Server-side
 
 If you have a :ref:`centralized API server <sky-api-server>` deployed, you can enforce a policy for all users by setting it at the server-side.
 
-First, install the Python package that implements the policy on the API server host:
+.. tab-set::
 
-.. code-block:: bash
+  .. tab-item:: Use a RESTful policy server
+  
+    Open the server's dashboard, go to :ref:`the server's SkyPilot config <sky-api-server-config>` and set the :ref:`admin_policy <config-yaml-admin-policy>` field to the API URL of the policy server.
+
+    .. code-block:: yaml
+
+        admin_policy: https://example.com/policy
+    
+  .. tab-item:: Use a Python package
+
+    First, install the Python package that implements the policy on the API server host:
+
+    .. code-block:: bash
+
+        pip install mypackage.subpackage
+
+    For helm deployment, refer to :ref:`sky-api-server-admin-policy` to install the policy package.
 
-    pip install mypackage.subpackage
+    Then, open the server's dashboard, go to :ref:`the server's SkyPilot config <sky-api-server-config>` and set the :ref:`admin_policy <config-yaml-admin-policy>` field to the path of the Python package that implements the policy.
 
-For helm deployment, refer to :ref:`sky-api-server-admin-policy` to install the policy package.
+    .. code-block:: yaml
 
-Then, open the server's dashboard, go to :ref:`the server's SkyPilot config <sky-api-server-config>` and set the :ref:`admin_policy <config-yaml-admin-policy>` field to the path of the Python package that implements the policy.
+        admin_policy: mypackage.subpackage.MyPolicy
 
-.. code-block:: yaml
+.. _serve-admin-policy-in-server:
+
+Serve admin policy in a RESTful server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    admin_policy: mypackage.subpackage.MyPolicy
+You can serve an admin policy in a RESTful API server and configure the SkyPilot to call the RESTful url to apply the policy.
+
+The server can be implemented in any language as long as it follows the API convention:
+
+.. dropdown:: The Admin Policy API
+
+    **POST /<api-path>**
+
+    Request body is a marshalled :ref:`sky.UserRequest <user-request-class>` in JSON format:
+
+    .. code-block:: json
```

Comment:
I feel we should show this after the python example, and all the user needs to do is to deal with our own dataclass instead of the raw json, ie they don't need to know the raw json format (we can leave json in a dropdown if they really want to do something on their own, in that case the return value needs to be limited)